doc/classes/EditorVCSInterface.xml

```diff
@@ -4,112 +4,301 @@
+			<argument index="0" name="file_path" type="String">
 			</argument>
 			<description>
-				Creates a version commit if the addon is initialized, else returns without doing anything. Uses the files which have been staged previously, with the commit message set to a value as provided as in the argument.
+				This method is called when user wants to a discard file.
 			</description>
 		</method>
-		<method name="get_file_diff">
+		<method name="_fetch" qualifiers="virtual">
+			<return type="void">
+			</return>
+			<description>
+				This method is called when user wants to fetch changes from remote.
+			</description>
+		</method>
+		<method name="_get_branch_list" qualifiers="virtual">
+			<return type="Array">
+			</return>
+			<description>
+				This method is called when editor needs list of branch. Returns Array of strings.
+			</description>
+		</method>
+		<method name="_get_file_diff" qualifiers="virtual">
+			<return type="Array">
+			</return>
+			<argument index="0" name="identifier" type="String">
+			</argument>
+			<argument index="1" name="area" type="int">
+			</argument>
+			<description>
+				Used to show file changes in bottom dock. The [code]identifier[/code] can be commit hash or file path based on value of area (see [enum TreeArea]). Returns Array of Dictionary (see [method create_diff_file] and [method add_diff_hunks_into_diff_file]).
+			</description>
+		</method>
+		<method name="_get_line_diff" qualifiers="virtual">
 			<return type="Array">
 			</return>
 			<argument index="0" name="file_path" type="String">
 			</argument>
+			<argument index="1" name="text" type="String">
+			</argument>
 			<description>
-				Returns an [Array] of [Dictionary] objects containing the diff output from the VCS in use, if a VCS addon is initialized, else returns an empty [Array] object. The diff contents also consist of some contextual lines which provide context to the observed line change in the file.
-				Each [Dictionary] object has the line diff contents under the keys:
-				- [code]"content"[/code] to store a [String] containing the line contents
-				- [code]"status"[/code] to store a [String] which contains [code]"+"[/code] in case the content is a line addition but it stores a [code]"-"[/code] in case of deletion and an empty string in the case the line content is neither an addition nor a deletion.
-				- [code]"new_line_number"[/code] to store an integer containing the new line number of the line content.
-				- [code]"line_count"[/code] to store an integer containing the number of lines in the line content.
-				- [code]"old_line_number"[/code] to store an integer containing the old line number of the line content.
-				- [code]"offset"[/code] to store the offset of the line change since the first contextual line content.
+				Used to show line change gutters in script editor. [b]Currently not used by editor.[/b] Returns Array of Dictionary (see [method create_diff_hunk]).
```

Comment:
```suggestion
				Returns an Array of Dictionary objects, each containing a line diff. (see [method create_diff_hunk])
```
I can't seem to recall what the `p_text` argument is for so feel free to add some words about it to this docstring